airflow/cli/commands/config_command.py

```diff
@@ -48,6 +49,14 @@ def show_config(args):
 
 def get_value(args):
     """Get one value from configuration."""
+    # while this will make get_value quite a bit slower we must initialize configuration
+    # for providers because we do not know what sections and options will be available after
+    # providers are initialized. Theoretically Providers might add new sections and options
+    # but also override defaults for existing options, so without loading all providers we
```

Comment:
Good point. I was thinkin about some aspects of it that might be good to dicuss. 

I will open a devlist dicussion on that as I think it might be something that we should at least discuss (and possibly decide) there.

Reply:
Discussion https://lists.apache.org/thread/lrlmsl7qz0rq3206zlybvf2yfzw29nby

Reply:
@jedcunningham -> saw your comment in slack. And yeah, agree that we shoudl "fail hard" if somoene attemps to override a section. Changing it.